Assets/_BForBoss/_Core/Scripts/Labor/RingSystem/RingLaborManager.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using BForBoss.Labor;
+using UnityEngine;
+using UnityEngine.Serialization;
+using Debug = UnityEngine.Debug;
+
+namespace BForBoss
+{
+    public class RingLaborManager : MonoBehaviour
```

Comment:
Don't need to change this for now. 

But in the future, This should be just named "LaborSystem" and everything below be named "Labor" related names. 

The point of this Labor is that it is generic and able to contain anything `Labor`.  So if we visually change the Rings to Lasers, it'll still be applicable. 
If we change the Rings into buttons, it'll still be applicable. 

The higher up the hierachy we go, the more abstract the managers should be, since the components lower down (the children) will be the more specific interchangeable components.

Reply:
Yeah I get that. This class has kinda a weird double purpose that I wasn't quite sure how best to address, where its serving both as a controller for the labor system and a view for setting up rings. Ideally I think those should be broken up

Reply:
We'll work with what we have here for now as the first iteration.